footprint/avr/run_suite.py

```
@@ -0,0 +1,231 @@
+KEY_VARIANTS = KEY_VARIANTS_FAST if FAST else KEY_VARIANTS_FULL
+TIMEOUT_RUN = 600  # seconds per simavr run (4096-bit AVR takes ~minute)
+TIMEOUT_BUILD = 600  # seconds for cargo build
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+TARGET_DIR = os.path.join(tempfile.gettempdir(), "rsa_footprint_avr")
+
+
+def run_cmd(args, timeout=TIMEOUT_RUN, **kwargs):
+    """Run a command, return (returncode, stdout, stderr)."""
+    env = os.environ.copy()
+    env.setdefault("CARGO_TARGET_DIR", TARGET_DIR)
+    result = subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)